finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -128,9 +128,16 @@ object HighResTimer {
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  val Default: com.twitter.util.Timer =
+  private[finagle] val Default: com.twitter.util.Timer =
     new JavaTimer(true, Some("HighResTimer"))
 
+  /**
+   * Stop default HighResTimer
+   */
+  def stop() {
```

Comment:
procedure-style syntax is being deprecated, please make it explicit:
```
def stop(): Unit = {...
```

Reply:
OK

Reply:
I think this actually hasn't been fixed yet.

Reply:
Done